src/cf-forms/usage.md

```diff
@@ -22,39 +22,44 @@ Capital Framework.
     - [Label helper text](#label-helper-text)
 - [Inputs](#inputs)
     - [Basic Text Inputs](#basic-text-inputs)
-        - [Default state](#default-state)
-        - [Hover state](#hover-state)
-        - [Focus state](#focus-state)
-        - [Disabled state](#disabled-state)
+        - [Text input default state](#text-input-default-state)
+        - [Text input hover state](#text-input-hover-state)
+        - [Text input focus state](#text-input-focus-state)
+        - [Text input disabled state](#text-input-disabled-state)
         - [Full-width modifier](#full-width-modifier)
     - [Basic checkboxes](#basic-checkboxes)
-        - [Default state](#default-state)
-        - [Hover/focus state](#hoverfocus-state)
-        - [Selected state](#selected-state)
-        - [Disabled state](#disabled-state)
+        - [Checkbox default state](#checkbox-default-state)
+        - [Checkbox hover state](#checkbox-hover-state)
+        - [Checkbox focus state](#checkbox-focus-state)
+        - [Checkbox selected state](#checkbox-selected-state)
+        - [Checkbox disabled state](#checkbox-disabled-state)
     - [Basic radio buttons](#basic-radio-buttons)
-        - [Default state](#default-state)
-        - [Hover/focus state](#hoverfocus-state)
-        - [Selected state](#selected-state)
-        - [Disabled state](#disabled-state)
+        - [Radio button default state](#radio-button-default-state)
+        - [Radio button hover state](#radio-button-hover-state)
+        - [Radio button focus state](#radio-button-focus-state)
+        - [Radio button selected state](#radio-button-selected-state)
+        - [Radio button disabled state](#radio-button-disabled-state)
     - [Large target area checkboxes](#large-target-area-checkboxes)
-        - [Default state](#default-state)
-        - [Hover/focus state](#hoverfocus-state)
-        - [Selected state](#selected-state)
-        - [Disabled state](#disabled-state)
+        - [Large checkbox default state](#large-checkbox-default-state)
+        - [Large checkbox hover state](#large-checkbox-hover-state)
+        - [Large checkbox focus state](#large-checkbox-focus-state)
+        - [Large checkbox selected state](#large-checkbox-selected-state)
+        - [Large checkbox disabled state](#large-checkbox-disabled-state)
     - [Large target area radio buttons](#large-target-area-radio-buttons)
-        - [Default state](#default-state)
-        - [Hover/focus state](#hoverfocus-state)
-        - [Selected state](#selected-state)
-        - [Disabled state](#disabled-state)
+        - [Large radio button default state](#large-radio-button-default-state)
```

Comment:
Same comment here - no need to repeat "Large radio button" for each...Could also add a "States" subheading and remove the word "state" from the things in the list.

Reply:
See comment above about links needing a unique name.